### ISSUE TITLE
Standardized template fields

### ISSUE THREAD

**Issue by jhauberg** [jhauberg/cards.py#68]
To support implementation of #60 and #56, I think the structure and handling of template fields could use an overhaul.

Essentially, I think it makes sense to make them more standardized, instead of working differently depending on the case: for example, image fields use ':' as a separator (e.g. `{{ my_image.png:16x16 }}`) for specifying image size, yet row references use a whitespace (e.g. `{{ my_column #7 }}`).

This should probably be standardized, so only one of the two is used. I think I prefer whitespace; however, that would likely clash with datasources having column names containing whitespace. But is that commonly done anyway? I know I wouldn't.

**Comment by jhauberg** [comment]
I think whitespace should be used as a general separator and simply just not be allowed in a column name. It should also be possible to issue multiple "commands" in a single field. For example, a field like `{{ my_title_column #7 uppercase }}` should resolve to the content of the **my_title_column** column in row 7, and finally uppercase the result.

I'm uncertain whether the order of commands should matter, or some commands just take priority; e.g. the `row` command is critical to occur before the `uppercase` command (and probably will be in any conceivable case?).

Image fields should be treated the same way; e.g. `{{ my_image.png 16x16 copy-only }}`

So to sum up: a template field is a _key_ and a list of _commands_. The left-most component, when separated by whitespace, is always the key and the rest are commands.

**Comment by jhauberg** [comment]
This stuff is in progress.

One thing, however; consider the field `{{ date uppercase }}`. What should happen here?

Right now, the field would attempt to use 'uppercase' as the format string, but what is clearly meant is to use the _command_ uppercase on the resulting date. Maybe there's a point to be made about splitting the context up into two pieces; variable parameters, and commands; e.g. taking a hint from Django or Jekyll:

```
{{ date }}
{{ date 'format' }}
{{ date 'format' | uppercase }}
{{ date | uppercase }}
```

and of course

```
{{ my_title_column #7 | uppercase }}
```
